RadarSDK/RadarUtils.m

```diff
@@ -32,7 +32,14 @@ + (NSNumber *)timeZoneOffset {
 }
 
 + (NSString *)sdkVersion {
-    return @"3.2.2";
+    NSBundle *bundle = [NSBundle bundleForClass:RadarUtils.class];
+    NSString *versionNumber = bundle.infoDictionary[@"CFBundleShortVersionString"];
+    
+    if (versionNumber == nil) {
+        return @"(unknown version)";
```

Comment:
@jrtibbetts @jsani-radar Are we validating the format of `sdkVersion` on the server today? (e.g., `x.y.z`) If so, this may fail. Should we: (a) update the validation, if it exists, (b) send `0.0.0` or some dummy value instead, or (c) throw an exception or log an error here because this should never happen?

Reply:
yes, we're doing validation of this field on the server, so this would fail - my preference would be to log an error so we get visibility in the logs feed + avoid crashing the app

Reply:
I can log an error, but what value should it return?